runelite-client/src/main/java/net/runelite/client/util/NonScheduledExecutorServiceExceptionLogger.java

```diff
@@ -0,0 +1,109 @@
+package net.runelite.client.util;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import static net.runelite.client.util.RunnableExceptionLogger.wrap;
+import org.jetbrains.annotations.NotNull;
+
+// Awkward name because plugins already referenced the ExecutorServiceExceptionLogger
+// (which only handles ScheduledExecutorServices) before this class was introduced
+public class NonScheduledExecutorServiceExceptionLogger implements ExecutorService
```

Comment:
Open to suggestions. Didn't want to rename the existing `ExecutorServiceExceptionLogger ` class (with targets a `ScheduledExecutorService`) because plugins reference it, and couldn't come up with anything better.